webview-ui/src/components/ViolationIncidentsList.tsx

```diff
@@ -138,6 +162,48 @@ const ViolationIncidentsList = ({
     </SelectList>
   );
 
+  const sortMenu = (
+    <Dropdown
+      isOpen={isSortOpen}
+      toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+        <MenuToggle
+          ref={toggleRef}
+          onClick={() => setIsSortOpen(!isSortOpen)}
+          isExpanded={isSortOpen}
+        >
+          {isSortAscending ? <SortAmountUpIcon /> : <SortAmountDownIcon />} Sort by{" "}
+          {sortField.charAt(0).toUpperCase() + sortField.slice(1)}
+        </MenuToggle>
+      )}
+      onSelect={() => setIsSortOpen(false)}
+    >
+      <DropdownItem key="none" onClick={() => handleSortSelect("none")}>
+        None
```

Comment:
I would change this to "default". Otherwise, the PR

Reply:
Moved to only ascending vs descending